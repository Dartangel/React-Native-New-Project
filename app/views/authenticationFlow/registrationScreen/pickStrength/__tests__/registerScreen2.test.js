import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PickStrength from '../pickStrength';

describe('PickStrength', () => {
    test('PickStrength snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<PickStrength />);
        expect(snapshot).toMatchSnapshot();
    });
});