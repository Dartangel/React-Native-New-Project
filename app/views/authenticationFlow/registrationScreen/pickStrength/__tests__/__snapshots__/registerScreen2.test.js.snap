// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PickStrength PickStrength snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Header
    header="How many times can you push out?"
    subHeader="Your data will help us make your workouts more effective."
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Picker
      itemSpace={25}
      itemStyle={null}
      onValueChange={[Function]}
      pickerData={
        Array [
          "0-5",
          "5-10",
          "10-20",
          "Более 20",
          "",
        ]
      }
      style={
        Object {
          "alignContent": "center",
          "backgroundColor": "white",
          "height": "65%",
          "paddingTop": 20,
          "width": "100%",
        }
      }
      textColor="#333"
      textSize={26}
    />
    <NextButton
      onPress={[Function]}
      title="Next"
    />
  </View>
</ImageBackground>
`;
